resources/web/wwi/X3dScene.js

```diff
@@ -14,6 +14,7 @@ export default class X3dScene {
   constructor(domElement) {
     this.domElement = domElement;
     this._loader = new Parser(this.prefix);
+    this.remainingRendering = 10; // Each time a render is needed, we ensure that there will be 10 additional renderings to avoid gtao artifacts
```

Comment:
```suggestion
    this.remainingRenderings = 10; // Each time a render is needed, we ensure that there will be 10 additional renderings to avoid gtao artifacts
```